pkg/compact/downsample/streamed_block_writer.go

```diff
@@ -113,7 +116,11 @@ func (w *streamedBlockWriter) WriteSeries(lset labels.Labels, chunks []chunks.Me
 	}
 
 	if len(chunks) == 0 {
-		level.Warn(w.logger).Log("msg", "empty chunks happened, skip series", "series", strings.ReplaceAll(lset.String(), "\"", "'"))
+		if w.logEmptyChunks {
+			level.Warn(w.logger).Log("msg", "empty chunks happened, skip series", "series", strings.ReplaceAll(lset.String(), "\"", "'"))
```

Comment:
I don't get the purpose of this parameter if it is always true. Why not just convert this line to `level.Debug`?

Reply:
is there any option to configure it through yaml ? or any config struct which can implement this ?

Reply:
the only way i  found was through the `NewStreamedBlockWriter` function which is called in `downsample.go`

Reply:
+1 for jus tusing level.Debug

Reply:
There's no way to control and it doesn't make sense to add a tunable parameter for this kind of minute thing. We already have way too many parameters and it is confusing for our users.

Reply:
or other options to fix this issue is by aggregating the warnings ? wdyt .. i think directly changing the level to debug is not an good option

Reply:
> There's no way to control and it doesn't make sense to add a tunable parameter for this kind of minute thing. We already have way too many parameters and it is confusing for our users.

yaa ! that's a point too .. so is there any need to change the level to debug?